control/settings.py

```diff
@@ -0,0 +1,55 @@
+#
+#  Copyright (c) 2021 International Business Machines
+#  All rights reserved.
+#
+#  SPDX-License-Identifier: LGPL-3.0-or-later
+#
+#  Authors: anita.shekar@ibm.com, sandy.kaur@ibm.com
+#
+
+import os
+import configparser
+import logging
+
+
+class Settings:
```

Comment:
I'm contemplating if we should use the same model as we do for the `PersistentConfig` and instead of having generic `get` functions have specific functions like `get_gateway_address` etc. This would keep all the config options in one file and would allow to sanitize the config according to the type. At least we should use constants defined in here to access the `Settings`. (As stated above let's make sure we use config vs settings consistently)

Reply:
I like how it would allow us to set defaults in one place. I'm for it. Can you expand on "At least we should use constants defined in here to access the `Settings`"? I'm not sure what you mean.

Reply:
I meant we should use constants for the config keys in the Settings class, e.g. GATEWAY_ADDRESS_KEY = gateway_addr etc.

Reply:
Moving further discussion of this to #49